language/stdlib/modules/Offer.move

```diff
@@ -35,6 +35,96 @@ module Offer {
   public fun address_of<Offered>(offer_address: address): address acquires T {
     borrow_global<T<Offered>>(offer_address).for
   }
+
+  /**
+   **************** SPECIFICATIONS ****************
+
+  This module defines a resource T that is used as a permissioned trading scheme between accounts.
+  It defines two main functions for creating and retrieving a struct offered by some user
+  inside the resource T under the offerer's account.
+
+  Currently, the only other module that depends on this module is LibraConfig, where it's used to
+  pass a capability to an account that allows it to modify a config.
+  */
+
+  /// # Module specification
+
+  spec module {
+    /// Verify all functions in this module
+    pragma verify = true;
+    /// Helper function that returns whether or not the `recipient` is an intended
+    /// recipient of the offered struct in the T<Offered> resource at the address `offer_address`
```

Comment:
`T<Offered>`

Reply:
I didn't understand this comment, can you elaborate please?

Reply:
Use backquote T<Offered> backquote ;-)

Reply:
... OMG... backquote T &lt; Offered &gt; backquote